app/precompiled.py

```diff
@@ -37,27 +37,39 @@
 FONT = "Arial"
 TRUE_TYPE_FONT_FILE = FONT + ".ttf"
 
-BORDER_FROM_BOTTOM_OF_PAGE = 5.0
-BORDER_FROM_TOP_OF_PAGE = 5.0
-BORDER_FROM_LEFT_OF_PAGE = 15.0
-BORDER_FROM_RIGHT_OF_PAGE = 15.0
-BODY_TOP_FROM_TOP_OF_PAGE = 95.00
+BORDER_FROM_BOTTOM_OF_PAGE = 5.0 - 1.0
```

Comment:
I think a code comment, probably here, to say what the +/- 1.0s are for generally in all the following variables. But I think I like the +/- 1.0 approach over say just changing the number to 4.0.